interview-questions.md

```diff
@@ -8,36 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: The foreign key can be added by adding a column called cohort_id:integer so that every student has a cohort and is able to be identified and referenced by that ID. Setting up the foreign Id helps link these two models to each other.
 
-Researched answer:
+Researched answer: The foreign ID is a attribute passed that refers to the primary key of a different model. It will link both models and you will be able to access information on both tables. Tables only allow one primary key so this a tool used to build a relationship between both tables. 
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+Your answer: Create, Show, Edit, Update, and Destroy. These are used to create HTTP requests to the server. 
 
-Researched answer:
+Researched answer: Parameters play an important part in communicating essential information the server needs to let it perform the specified task accurately. They provide extra information to the server so they can be used in a multitude of ways and can also modify the behavior of the API endpoint. 
```

Comment:
The routes would be show, edit, update, destroy. These RESTful routes are targeting one specific primary key and performing their action on only the targeted id.